.github/issue_template.md

```diff
@@ -1,12 +1,14 @@
-### IMPORTANT ###
-This is not a support forum, it's a bug tracker. If you don't understand the difference, please do NOT open an issue. For questions and support, review recent postings on the john-users mailing list at https://www.openwall.com/lists/john-users/ and then subscribe to the list at https://www.openwall.com/john/#lists and post your message to the list by e-mailing `john-users at lists.openwall.com`.
-
-We are not interested in bugs in your distro-supplied and/or several years old Jumbo. Please do not open an issue unless you tried the latest version from HERE first.
+### Checklist ###
+- [ ] :1st_place_medal: I've read and understood these instrucions;
+  - This is not a support forum, it's a bug tracker. For questions and support, review postings on the [john-users mailing list](https://www.openwall.com/lists/john-users/).
+- [ ] :thumbsup: I've tested using latest bleeding version from this repository.
+  - Be clear about your environment and what you are doing. Share a sample hash or file that can be used to reproduce.
+- [ ] :confused: I'm confused and I need guidance.
+  - Please, post your message to john-users list by e-mailing `john-users at lists.openwall.com`.
```

Comment:
I think we need people to read the instructions at https://www.openwall.com/john/#lists and to join the list before posting. Asking anyone confused to just go ahead and post will increase noise on the list, and will also result in people posting to the public list without realizing they're making the information public, then complain and ask to remove stuff from list archives.